### DISCUSSION TITLE
About Audio Quality

### DISCUSSION THREAD

**Discussion by qinzheng07** [ciderapp/Cider#1108] [Q&A] [answered]
I have a question about audio quality。I went to youtube and saw that there is an option for extreme in the audio quality option, but only low and high options are available on the version I installed.Excuse me, how can I improve the playback sound quality of cider?My English is not good, I rely on Google to translate, please forgive me. thank you very much!
![image](https://user-images.githubusercontent.com/106638276/171524874-d0022179-db56-4001-99be-bfce399654b9.png)
The picture above is from youtube.

**Comment by Amaru8** [comment, marked as answer]
**Hi @qinzheng07!** Lossless playback is not currently supported in Cider. Apple's MusicKit Framework does have lossless support, however, decryption of this audio is not supported.

**Reply by marcell1no** [reply]
Can you guys estimate a timeframe for a solution to this?

**Reply by Amaru8** [reply]
**No**, because this fully depends on Apple.
